x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/rules/RuleQueryBuilderTests.java

```diff
@@ -175,4 +172,20 @@ protected Object simulateMethod(Method method, Object[] args) {
 
         return super.simulateMethod(method, args);
     }
+
+    /**
+     * Returns a map where the keys are object names that won't trigger a standard exception (an exception that contains the string
```

Comment:
Do we need to copy the comment? It's on the superclass so it can be checked there...

Reply:
You're right, I can remove it!